### DISCUSSION TITLE
Sonen Dashboard visualisation issue

### DISCUSSION THREAD

**Discussion by doncuco** [sonnen-tools/node-red#11] [General]
Hey,

one question: is it correct, that the sonnen dashboard is not showing the charging process of the battery correctly?
So what this show us here is the House consumption and not the grid in value.

<img width="813" alt="image" src="https://github.com/sonnen-tools/node-red/assets/122402791/d27a22dc-49dc-43e7-a200-168e0e33b4ba">

The dashboard directly on the battery is showing this correct:

<img width="399" alt="image" src="https://github.com/sonnen-tools/node-red/assets/122402791/d37199d2-6ff7-4a23-8298-5e02c586ee1b">

But the graphs are missing the charging from grid values:

<img width="399" alt="image" src="https://github.com/sonnen-tools/node-red/assets/122402791/7eba44a7-8782-42d5-a9af-ffabf8e1d291">

So you see the same on your board?

Thanks.

**Comment by markusjnagel** [comment]
Yes, same here - Sonnen doesn't show it as consumption (although it is available in the API - and the local dashboard, as you have mentioned). 

Last night example (I set the target SoC to 97% for the cheap hours):

No consumption while charging the battery (if only that was true :laughing: )
![image](https://github.com/sonnen-tools/node-red/assets/33287465/3ce053b5-65b7-43b7-a90b-0c2383c76511)
